src/svymean.jl

```diff
@@ -84,15 +84,47 @@ function sem_svyby(x::AbstractVector, design::SimpleRandomSample, _)
 end
 
 """
-Inner method for `svyby`.
+Inner method for `svyby` to calculate standard error of domain mean of StratifiedSample.
 """
-function svymean(x::AbstractVector, design::SimpleRandomSample, weights)
```

Comment:
It looks like you removed the `svymean` method for `svyby` for `SimpleRandomSample` and you didn't replace it with anything else. Why did you remove it? It was working fine for SRS.

Reply:
Sorry, I saw now that you just moved it a few lines below. The Stratified and SRS functions got mixed up. Try to fix that.

Reply:
Yep its still there